haskell-org-committee.markdown

```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
-
 Committee members do not need to be deep technical Haskell experts.  Instead we look for enthusiasm towards improving the Haskell community.  We aim to represent the different facets of the community.  We aim to be diverse in terms of industry or research, and in terms of gender, race and location.
 
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
```

Comment:
I think this PDF should be imported/added here to the site, and hosted here / not lost.

Reply:
Good call. I figure we can just add it to the repo.

If you're not sure where to put it so that it's available as a download, I can give it a try—I don't know myself off-hand, so I'd have to poke around the code a bit.

Reply:
I've gotten that worked out, I think.